onebusaway-android/src/main/java/org/onebusaway/android/ui/ArrivalsListFragment.java

```diff
@@ -1638,6 +1639,23 @@ public boolean equals(Object obj) {
         }
     }
 
+    private void reportAnalytics(ObaSituation situation) {
+        ObaSituation.AllAffects[] allAffects = situation.getAllAffects();
+        List<String> agencyIds = new ArrayList<>();
+        for (int i=0; i< allAffects.length; i++) {
```

Comment:
Minor formatting issue - let's change to:

`for (int i = 0; i < allAffects.length; i++) {`